OpenProblemLibrary/macros/Alfred/Alfredmacros.pl

```diff
@@ -27,20 +27,22 @@ sub inversetrig
 }
 
 ## Compute the max and min of an array of numbers
-sub max {
-my $maximum = shift @_;
-foreach(@_){
-   if ($maximum < $_){$maximum = $_};
-};
-return $maximum;
-} 
-sub min {
-my $minimum = shift @_;
-foreach(@_){
-   if ($minimum > $_){$minimum =  $_};
-};
-return $minimum;
-} 
+## Deleted (max and min are provided by PGauxiliaryFunctions.pl,
+## which is loaded by PGstandard.pl in every problem).
+#sub max {
+#my $maximum = shift @_;
+#foreach(@_){
+#   if ($maximum < $_){$maximum = $_};
+#};
+#return $maximum;
+#}
+#sub min {
+#my $minimum = shift @_;
+#foreach(@_){
+#   if ($minimum > $_){$minimum =  $_};
+#};
+#return $minimum;
+#}
```

Comment:
```suggestion
```

Reply:
Simply delete these lines. WeBWorK has a bad habit of retaining unnecessary comments... one can always review the git history to retrieve old code, if it's ever relevant.